crates/dbsp/src/operator/communication/exchange.rs

```diff
@@ -512,7 +522,7 @@ where
                                     .unwrap()
                                     .take()
                                     .unwrap();
-                                to_bytes(&item).unwrap().into_vec()
+                                rkyv_serialize(&item).unwrap().into_vec()
```

Comment:
looks like we're running this under TOKIO runtime right now, might be worth to use monoio here too, since it avoids mixing two async runtime (one that can move tasks across cores, one that can't)

Reply:
Curious why we need to call into_vec() here since it's already a buffer

Reply:
I think I couldn't figure out how to send an `FBuf` (or an `AlignedVec`) over a `tarpc` channel.

`tarpc` isn't very special. When we get back to working on distribution, it might make sense to use something else instead.